allennlp/nn/util.py

```diff
@@ -507,16 +507,16 @@ def flatten_and_batch_shift_indices(indices: torch.Tensor,
     correctly indexes into the flattened target. The sequence length of the target must be
     provided to compute the appropriate offsets.
 
-    example:
-    .. code-block:: python
-        indices = torch.ones([2,3]).long()
-        # Sequence length of the target tensor.
-        sequence_length = 10
-        shifted_indices = flatten_and_batch_shift_indices(indices, sequence_length)
-        # Indices into the second element in the batch are correctly shifted
-        # to take into account that the target tensor will be flattened before
-        # the indices are applied.
-        assert shifted_indices == [1, 1, 1, 11, 11, 11]
+    ```
```

Comment:
Are you sure this works?  I think the `code-block` thing you had is right, it just maybe didn't like the `example:`?